README.md

```diff
@@ -24,12 +24,14 @@ the application is being run. This is best practice as described by
 
 Start by cloning the repo and all submodules i.e. `git clone https://github.com/nhsuk/gp-finder.git && cd gp-finder/ && git submodule update --init --recursive`
 
-Run the application with Docker via `docker-compose up --build --force-recreate`. This will build an image based on the code in the current working directory and start it running. It will most likely be available locally on http://localhost:3000 as long as the port exposed in `docker-compose.yml` is `3000`.
+You can run the application with Docker via `docker-compose up --build --force-recreate`. This will build an image based on the code in the current working directory and start it running. It will most likely be available locally on http://localhost:3000 as long as the port exposed in `docker-compose.yml` is `3000`.
```

Comment:
Would also be good to decide which scripts are we going with the ones here in `gp-finder` or the ones in `profiles` before we write down how they work/document them?